src/color-scale/color-scale.js

```diff
@@ -71,6 +71,9 @@ const Self = class ColorScale extends NudeElement {
 			if (this.coords) {
 				swatch.coords = this.coords;
 			}
+			if (this.vs) {
+				swatch.vs = this.vs;
+			}
```

Comment:
We don’t want to copy this into the swatch. The scale knows what the previous color is!! All we need is to signify that we want to show deltas

Reply:
I suppose showing the delta from a specific color is useful, but I’d expect the primary use case is to show deltas within the scale

Reply:
So, we should do this, right?

```js
if (i > 0) {
	swatch.vs = colors[i - 1].color;
}
```

Reply:
And we should also support an attribute for signify that we want to show deltas.

Reply:
Something like this:

```js
deltas: {
	type: Boolean,
},
```

```js
if (this.deltas) {
	if (this.vs) {
		swatch.vs = this.vs;
	}
	else if (i > 0) {
		swatch.vs = colors[i - 1].color;
	}
}
```

Reply:
I committed the change that should support the primary use case.